packages/shared/graphql/generated/GetTestsForWorkspace.ts

```diff
@@ -11,19 +11,11 @@ export interface GetTestsForWorkspace_node_Recording {
   __typename: "Recording";
 }
 
-export interface GetTestsForWorkspace_node_Workspace_tests_edges_node_executions {
-  __typename: "TestExecution";
-  errors: string[] | null;
-  createdAt: any;
-  result: string;
-}
-
 export interface GetTestsForWorkspace_node_Workspace_tests_edges_node {
```

Comment:
> it splits the work into two separate queries. `GetTestsForWorkspace` which is responsible for getting the test names, and `GetTestForWorkspace` which is responsible for getting the execution/replay data for the selected workspace.

If this is just used to get test names then it shouldn't be named "get tests" because that's confusing.

Reply:
Renamed this to `GetTestPreviewsForWorkspace` for now to keep them separate!